lib/temporal/configuration.rb

```diff
@@ -75,26 +76,32 @@ def timeouts=(new_timeouts)
       @timeouts = DEFAULT_TIMEOUTS.merge(new_timeouts)
     end
 
-    def converter
-      @converter
-    end
+    attr_reader :converter
```

Comment:
nit: belongs with other `attr_reader`s higher up

Reply:
Good catch. We've started running [rubyfmt](https://twitter.com/defrag_fable/status/1577293122245201921) on any Ruby code automatically. It's even clever enough to make modifications like this, but I guess not to group the attributes together.